apiserver/facades/client/application/application_unit_test.go

```diff
@@ -507,7 +507,7 @@ func (s *ApplicationSuite) assertDestroyApplication(c *gc.C, force bool) {
 		"UnitStorageAttachments",
 		"ApplyOperation",
 	)
-	s.backend.CheckCall(c, 7, "ApplyOperation", &state.DestroyApplicationOperation{Force: force})
+	s.backend.CheckCall(c, 7, "ApplyOperation", &state.DestroyApplicationOperation{ForcedOperation: &state.ForcedOperation{Force: force}})
```

Comment:
Is this right? I thought you changed it to not be a pointer. Maybe it works because of the way `.CheckCall` compares the call structs?